test-harness/src/test/java/com/netflix/conductor/tests/integration/End2EndTests.java

```diff
@@ -203,7 +212,37 @@ public void testAll() throws Exception {
 		assertNotNull(wf);
 		assertEquals(WorkflowStatus.RUNNING, wf.getStatus());
 		assertEquals(1, wf.getTasks().size());
-		
+
+	}
+
+	@Test
+	public void testMetadataWorkflowDefinition() {
+		WorkflowDef def = new WorkflowDef();
+		def.setName("testWorkflowDel");
+		def.setVersion(1);
```

Comment:
Question - Let's say we have 5 versions of workflows, can I delete in-between versions? I.e [1,2,3,4] etc.